garble/mpz-garble/src/lib.rs

```diff
@@ -361,30 +362,30 @@ pub trait Verify {
         outputs: &[ValueRef],
     ) -> Result<(), VerifyError>;
 
-    /// Verifies the provided values against the expected values.
+    /// Verifies the provided output values against the expected values.
     async fn verify(
         &mut self,
         values: &[ValueRef],
         expected_values: &[Value],
     ) -> Result<(), VerifyError>;
 }
 
-/// This trait provides methods for decoding values.
+/// This trait provides methods for decoding output values.
```

Comment:
I can't think of any concrete scenarios where there would be a need to decode input values. I think we should try to keep the comments less abstract where possible (it will cause less abstraction fatigue for the reader).  Later, when there is a real use-case for using these traits on input values, we can modify the comments accordingly.

Reply:
ah, ok, thx, goot to know. then i'll revisit this PR in light of that PR later on.